.github/ISSUE_TEMPLATE/translation.yml

```
@@ -0,0 +1,35 @@
+name: Translation
+description: Request a translation
+title: ":twisted_rightwards_arrows: "
```

```suggestion
title: ":twisted_rightwards_arrows: "
```
I had been using 🌐  here, which may be nice to keep consistent, unless you feel strongly about this emoji!

---

I don't have a preference at all! I'll update this as well.